backend/src/uxbox/services/queries/recent_files.clj

```diff
@@ -0,0 +1,58 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; This Source Code Form is "Incompatible With Secondary Licenses", as
+;; defined by the Mozilla Public License, v. 2.0.
+;;
+;; Copyright (c) 2019-2020 Andrey Antukh <niwi@niwi.nz>
+
+(ns uxbox.services.queries.recent-files
+  (:require
+   [clojure.spec.alpha :as s]
+   [promesa.core :as p]
+   [uxbox.db :as db]
+   [uxbox.common.spec :as us]
+   [uxbox.services.queries :as sq]
+   [uxbox.services.queries.projects :refer [ projects-by-team ]]
+   [uxbox.services.queries.files :refer [ decode-row ]]))
+
+(def ^:private sql:project_files_recent
```

Comment:
I think `-` should be used instead of `_`

Reply:
OK